client/src/components/Categories/SingleCategory.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import apiClient from '../../helpers/apiClient'
+
+class SingleCategory extends Component {
+
+    constructor() {
+
+        super();
+        this.state = {
+            articles: []
+        }
+    }
+    componentDidMount() {
+        apiClient.getArticles()
+            .then(({ data }) => {
+                this.setState({
+                    articles: data
+                })
+            })
+            .catch((err) => { })
```

Comment:
never have an empty catch. Remove it - it's better if the application crashes on an error, so that we fix it, than it goes unnoticed.